docs/introduction-to-smart-contracts.rst

```diff
@@ -224,8 +224,8 @@ a failure can more easily be debugged or reacted upon.
 
 The ``send`` function can be used by anyone (who already
 has some of these coins) to send coins to anyone else. If the sender does not have
-enough coins to send, the ``require`` call fails and provides the
-sender with an appropriate error message string.
+enough coins to send, the ``if`` statement evaluates to true. As a result, the operation will fail
+while providing the sender with an appropriate error message.
```

Comment:
Thanks for noticing! Can you somehow mention "revert statement" in this sentence? Otherwise, people might think that the `if` causes the failure.

Reply:
Sure!